tests/Rector/CodeQuality/ForeachToInArrayRector/Correct/correct.php.inc

```diff
@@ -0,0 +1,9 @@
+<?php declare(strict_types=1);
+
+final class MyClass
+{
+    public function foreachToInArray(): bool
```

Comment:
Strange @TomasVotruba, now this return type is been affected for some reason:

```sh
carusogabriel@caruso:~/Code/rector$ vendor/bin/phpunit tests/Rector/CodeQuality/ForeachToInArrayRector/ForeachToInArrayRectorTest.php
PHPUnit 7.4.0 by Sebastian Bergmann and contributors.

Runtime:       PHP 7.2.10-0ubuntu0.18.04.1
Configuration: /home/carusogabriel/Code/rector/phpunit.xml

F                                                                   1 / 1 (100%)

Time: 83 ms, Memory: 16.00MB

There was 1 failure:

1) Rector\Tests\Rector\CodeQuality\ForeachToInArrayRector\ForeachToInArrayRectorTest::test with data set #0 ('/home/carusogabriel/Code/rect...hp.inc', '/home/carusogabriel/Code/rect...hp.inc')
Original file "/home/carusogabriel/Code/rector/tests/Rector/CodeQuality/ForeachToInArrayRector/Wrong/wrong.php.inc" did not match the result.
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
 \n
 final class MyClass\n
 {\n
-    public function foreachToInArray(): bool\n
+    public function foreachToInArray() : bool\n
     {\n
         return in_array('something', $items, true);\n
     }\n
 }\n
 '

/home/carusogabriel/Code/rector/src/Testing/PHPUnit/AbstractRectorTestCase.php:84
/home/carusogabriel/Code/rector/tests/Rector/CodeQuality/ForeachToInArrayRector/ForeachToInArrayRectorTest.php:18
```

Reply:
Hm :thinking: , I'm not sure about this.
It's job of Printer. You can try nulling nodes, that are used to reconstruct the style.

```php
$node->setAttribute(Attribute::PARENT_NODE, 'null');
$node->setAttribute(Attribute::ORIGINAL_NODE, 'null')
```

Reply:
It's looks really good though. I have few ideas how to extends it.
Found many places in my code where this will be useful :)

Reply:
@TomasVotruba I got no success with this. I'll finish this PR without this return types. Can you please report this to PHP-Parser? You'll know better than me what part to report :)

Reply:
No troubles :+1: . Feel free to merge it, I'll check it locally then